test/command/suite/sharding/logical_range_filter/columns/stage/filtered/window_function/month/min.expected

```diff
@@ -70,9 +70,22 @@ logical_range_filter Logs   --shard_key timestamp   --min '2017/03/15 01:00:00'
     ]
   ]
 ]
+#|-| [obj][open] <273>(<Logs_201703.timestamp>):<64>(<column:fix_size>)
+#|-| [obj][open] <272>(<Logs_201703>):<51>(<table:no_key>)
```

Comment:
An opened table, `Logs_201703` in this case, is not closed in cases below.

1. using `time_classify_xxx` with dynamic column
2. using `window` function with dynamic column
3. specifying a non existent column with `output_column`

`select` command also has these issues so I guess this is not the matter of `logical_range_filter`.
The commands to reproduce these are below.


**common**
```
plugin_register functions/time
table_create Logs_20170315 TABLE_NO_KEY
column_create Logs_20170315 timestamp COLUMN_SCALAR Time
column_create Logs_20170315 price COLUMN_SCALAR UInt32
load --table Logs_20170315
[
{"timestamp": "2017/03/15 00:00:00", "price": 1000},
{"timestamp": "2017/03/15 11:00:00", "price":  900},
{"timestamp": "2017/03/15 12:00:00", "price":  300},
{"timestamp": "2017/03/15 13:00:00", "price":  200}
]
log_level --level dump
```


**Case 1: using `time_classify_xxx` with dynamic column**
```
select Logs_20170315   --columns[day].stage filtered   --columns[day].type Time   --columns[day].flags COLUMN_SCALAR   --columns[day].value 'time_classify_day(timestamp-36000000000)'   --output_columns _id
[[0,0.0,0.0],[[[4],[["_id","UInt32"]],[1],[2],[3],[4]]]]
#|-| [obj][open] <272>(<Logs_20170315>):<51>(<table:no_key>)
#|-| [obj][open] <273>(<Logs_20170315.timestamp>):<64>(<column:fix_size>)
#|-| [obj][close] <273>(<Logs_20170315.timestamp>):<64>(<column:fix_size>)
```

**Case 2: using `window` function with dynamic column**
```
select Logs_20170315   --columns[price_per_day].stage filtered   --columns[price_per_day].type UInt32   --columns[price_per_day].flags COLUMN_SCALAR   --columns[price_per_day].value 'window_sum(price)'   --columns[price_per_day].window.group_keys 'timestamp'   --output_columns _id
[[0,0.0,0.0],[[[4],[["_id","UInt32"]],[1],[2],[3],[4]]]]
#|-| [obj][open] <272>(<Logs_20170315>):<51>(<table:no_key>)
#|-| [obj][open] <274>(<Logs_20170315.price>):<64>(<column:fix_size>)
#|-| [obj][open] <273>(<Logs_20170315.timestamp>):<64>(<column:fix_size>)
#|-| [obj][close] <274>(<Logs_20170315.price>):<64>(<column:fix_size>)
#|-| [obj][close] <273>(<Logs_20170315.timestamp>):<64>(<column:fix_size>)
```

**Case 3: specifying a non existent column with `output_column`**
```
select Logs_20170315   --filter 'price >= 300'   --output_columns _id,non_existant
[[0,0.0,0.0],[[[3],[["_id","UInt32"]],[1],[2],[3]]]]
#|-| [obj][open] <272>(<Logs_20170315>):<51>(<table:no_key>)
#|-| [obj][open] <274>(<Logs_20170315.price>):<64>(<column:fix_size>)
#|-| [obj][close] <274>(<Logs_20170315.price>):<64>(<column:fix_size>)
```

Reply:
> so I guess this is not the matter of `logical_range_filter`.

Because of this, I will create a new PR to fix this.
I think this PR should focus on `logical_range_filter`.

Reply:
It makes sense.

Reply:
> * using `time_classify_xxx` with dynamic column
> 
> * using `window` function with dynamic column
> 
> * specifying a non existent column on `output_column` with specifying `filter`

Those cases are also fixed by the change of `db.c`.

Reply:
Could you split it to another pull request as you said?

Reply:
However,  `Logs_201703`  is still not be closed at this test case. I am investigating it.

Reply:
> Could you split it to another pull request as you said?

I see. I will create a new PR.

Reply:
I have rebased onto `master` and updated tests and confirmed it is closed.